<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>RGLAB::PUBLICATIONS</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 66px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 71px;
  margin-top: -71px;
}

.section h2 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h3 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h4 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h5 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h6 {
  padding-top: 71px;
  margin-top: -71px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">RGLAB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-pencil"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="software.html">
    <span class="fa fa-laptop"></span>
     
    Software
  </a>
</li>
<li>
  <a href="members.html">
    <span class="fa fa-user"></span>
     
    Members
  </a>
</li>
<li>
  <a href="contact.html">
    <span class="fa fa-envelope"></span>
     
    Contact
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://twitter.com/raphg">
    <span class="fa fa-twitter"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/RGLab">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">RGLAB::PUBLICATIONS</h1>

</div>


<div id="section" class="section level2">
<h2>2015</h2>
<ul>
<li>Florian Hladik, Adam Burgener, Lamar Ballweber, Raphael Gottardo, Slim Fourati, James Y. Dai, Mark J. Cameron, Lucia Vojtech, Johanna Strobl, Sean Hughes, M.A., Craig Hoesley, Ross Cranston, Philip Andrew, Jeanna Piper, T. Blake Ball, Gustavo F. Doncel, David R. Friend, Kenneth Mayer, M. Juliana McElrath, and Ian McGowan. Systems Biology Reveals Significant Mucosal Side Effects of Tenofovir 1% Gel in a Phase I Rectal Microbicide Trial. eLife, In Press.</li>
<li>Lin Lin, Greg Finak, Kevin Ushey, Chetan Seshadri, Thomas R. Hawn, Nicole Frahm, Thomas J. Scriba, Hassan Mahomed, Willem Hanekom, Pierre-Alexandre Bart, Giuseppe Pantaleo, Georgia Tomaras, Supachai Rerks-Ngarm, Jaranit Kaewkungwal, Sorachai Nitayanphan, Punnee Pitisuttithum, Nelson L. Michael, Jerome H. Kim, Merlin L. Robb, Robert J. O’Connell, Nicos Karasavvas, Bart Haynes, Peter Gilbert, Stephen DeRosa, Julie McElrath and Raphael Gottardo. Combinatorial Polyfunctionality Analysis of Antigen-Specific T-cell Subsets Identifies Novel Cellular Subsets Correlated with Clinical Outcomes. Nature Biotechnology, In Press.</li>
<li>Carey V, Carlson M, Corrada Bravo H, Carvalho BS, Davis S, Gatto L, Gentleman R, Girke T., Gottardo R., Hahne F., Hansen K., Huber W., Irizarry R. A., Lawrence M., MacDonald J, Obenchain V, Koles A, Pages H, Shannon P, Tenenbaum D, Waldron L, Morgan M. Orchestrating high-throughput genomic analysis with Bioconductor. Nature Methods, In Press.</li>
<li>Lin Lin, Jacob Frelinger, Mike Jiang, Greg Finak, Chetan Seshadri, Pierre-Alexandre Bart, Giuseppe Pantaleo, Julie McElrath, Steve DeRosa, and Raphael Gottardo. Identification and visualization of multidimensional antigen-specific T-cell populations in polychromatic cytometry data. Cytometry A, In Press.</li>
<li>Malek M, Taghiyar MJ, Chong L, Finak G, Gottardo R, Brinkman RR. flowDensity: reproducing manual gating of flow cytometry data by automated density-based cell population identification. Bioinformatics. 2015, 10.1093/bioinformatics/btu677. Epub ahead of print. PMID: 25378466.</li>
<li>Courtot M, Meskas J, Diehl AD, Droumeva R, Gottardo R, Jalali A, Renani JT, Maecker HT, McCoy JP, Ruttenberg A, Scheuermann RH, Brinkman RR. flowCL: ontology-based cell population labelling in flow cytometry. Bioinformatics. 2015, 10.1093/bioinformatics/btu807. Epub ahead of print. PMID: 25481008.</li>
<li>Paul T. Edlefsen, Morgane Rolland, Tomer Hertz, Andrew Gartland, Allan C. deCamp, Craig A. Magaret, Hasan Ahmed, Raphael Gottardo, Michal Juraska, Frederick A. Matsen IV, Connor McCoy, Brendan B. Larsen, Sodsai Tovanabutra, Eric Sanders-Buell, Chris Carrico, Sergey Menis, Lennie Chen, Philip Konopa, Snehal Nariya, Julia N. Stoddard, Kim Wong, Hong Zhao, Wenjie Deng, Brandon S. Maust, Meera Bose, Shana Howell, Adam Bates, Michelle Lazzaro, Annemarie O’Sullivan, Esther Lei, Andrea Bradfield, Grace Ibitamuno, Vatcharain Assawadarachai, Robert J. O’Connell, Mark S. deSouza, Sorachai Nitayaphan, Supachai Rerks-Ngarm, Merlin L. Robb, Jason S. McLellan, Ivelin Georgiev, Peter D. Kwong, Jonathan M. Carlson, Nelson L. Michael, William R. Schief, James I. Mullins, Jerome H. Kim, and Peter B. Gilbert. Comprehensive sieve analysis of breakthrough HIV-1 sequences in the RV144 vaccine efficacy trial. PLoS Computational Biology, In Press.</li>
<li>Ann Dahlberg, Sangsoon Woo, Colleen Delaney, Patrick Boyle, Andreas Gnirke, Christoph Bock, Bradley Bernstein, Alex Meissner, Raphael Gottardo, and Irwin Bernstein. Notch-mediated expansion of cord blood progenitors: maintenance of transcriptional and epigenetic fidelity. Leukemia, In Press.</li>
</ul>
</div>
<div id="section-1" class="section level2">
<h2>2014</h2>
<ul>
<li>Lemos MP, Lama JR, Karuna ST, Fong Y, Montano SM, Ganoza C, Gottardo R, Sanchez J, McElrath MJ. The inner foreskin of healthy males at risk of HIV infection harbors epithelial CD4+ CCR5+ cells and has features of an inflamed epidermal barrier. PLoS One. 2014;9(9):e108954. PMCID: PMC4182607.</li>
<li>Zolla-Pazner S, Edlefsen PT, Rolland M, Kong X, deCamp A, Gottardo R, Williams C, Tovanabutra S, Mullins JI, deSouza MS, Karasavvas N, Nitayaphan S, Rerks-Ngarm S, Pitisuttithum P, Kaewkungwal J, O’Connell RJ, Robb ML, Michael NL, Kim JH, Gilbert PB. Vaccine-induced human antibodies specific for the third variable region of HIV-1 gp120 impose immune pressure on infecting viruses. eBioMedicine&gt;. 2014;1(1):37-45. PMCID: PMC4293639.</li>
<li>Finak G, Frelinger J, Jiang W, Newell EW, Ramey J, Davis MM, Kalams SA, De Rosa SC, Gottardo R. OpenCyto: An Open Source Infrastructure for Scalable, Robust, Reproducible, and Automated, End-to-End Flow Cytometry Data Analysis. PLoS Computational Biology, In Press.</li>
<li>McDavid A, Dennis L, Danaher P, Finak G, Krouse M, Wang A, Webster P, Beechem J, Gottardo R. Modeling bi-modality improves characterization of cell cycle on gene expression in single cells. PLoS Computational Biology, In Press.</li>
<li>Li SS, Gilbert PB, Tomaras GD, Kijak G, Ferrari G, Thomas R, Pyo C-W, Zolla-Pazner S, Montefiori D, Liao H-X, Nabel G, Pinter A, Evans DT, Gottardo R, Dai JY, Janes H, Morris M, Fong Y, Edlefsen PT, Li F, Frahm N, Alpert MD, Prentice H, Rerks-Ngarm S, Pitisuttithum P, Kaewkungwal J, Nitayaphan S, Robb ML, O’Connell RJ, Haynes BF, Michael NL, Kim JH, McElrath MJ, Geraghty DE. Association of FCGR2C polymorphisms with vaccine efficacy and correlates of HIV-1 infection risk in the RV144 trial. Journal of Clinical Investigation, In Press.</li>
<li>Gartland A, Li S, McNevin J, Tomaras GD, Gottardo R, Janes H, Fong Y, Morris D, Geraghty DE, Kijak GH, Edlefsen PT, Frahm F, Larsen BB, Tovanabutra S, Sanders-Buell E, deCamp AC, Magaret CA, Ahmed H, Goodridge JP, Chen L, Konopa P, Nariya S, Stoddard JN, Wong K, Zhao H, Deng W, Maust BS, Bose M, Howell S, Bates A, Lazzaro M, O’Sullivan A, Lei E, Bradfield A, Ibitamuno G, Assawadarachai V, O’Connell RJ, deSouza MS, Nitayaphan S, Rerks-Ngarm S, Robb ML, Sidney J, Sette A, Zolla-Pazner S, Montefiori DC, McElrath MJ, Mullins JI, Kim JH, Gilbert PB, Hertz T. Analysis of HLA A*02 association with vaccine efficacy in the RV144 Thai trial. Journal of Virology. 2014;88(15):8242-55.</li>
<li>Roederer M, Keele BF, Schmidt SD, Mason RD, Welles HC, Fischer W, Labranche C, Foulds KE, Louder MK, Yang ZY, Todd JP, Buzby AP, Mach LV, Shen L, Seaton KE, Ward BM, Bailer RT, Gottardo R, Gu W, Ferrari G, Alam SM, Denny TN, Montefiori DC, Tomaras GD, Korber BT, Nason MC, Seder RA, Koup RA, Letvin NL, Rao SS, Nabel GJ, Mascola JR. Immunological and virological mechanisms of vaccine-mediated protection against SIV and HIV. Nature. 2014;505:502-8.</li>
<li>Vojtech L, Woo S, Hughes S, Levy C, Ballweber L, Sauteraud RP, Strobl J, Westerberg K, Gottardo R, Tewari M, Hladik F. Exosomes in human semen carry a distinctive repertoire of small non-coding RNAs with potential regulatory functions. Nucleic Acids Research. 2014;42(11):7290-304.</li>
<li>Finak G, Jiang M, Krouse K, Phippard D, Asare A, DeRosa SC, Self S, Gottardo R. High throughput flow cytometry data normalization for clinical trials. Cytometry A, 2014;85(3):277-86.</li>
<li>Brusic* V., Gottardo R.<em>, Kleinstein S.H., Davis M.M., and the HIPC steering committee. Computational resources for high-dimension immune analysis from the Human Immunology Project Consortium. Nature Biotechnology, 2014;32(2):146-8. </em>Equal contribution</li>
<li>deCamp A, Hraber P, Bailer RT, Seaman MS, Ochsenbauer C, Kappes J, Gottardo R, Edlefsen P, Self S, Tang H, Greene K, Gao H, Daniell X, Sarzotti-Kelsoe M, LaBranche C, Mascola JR, Korber BT, Monteﬁori DC. Global Panel of HIV-1 Env Reference Strains for Standardized Assessments of Vaccine-Elicited Neutralizing Antibodies. Journal of Virology, 2014;88(5):2489-507.</li>
</ul>
</div>
<div id="section-2" class="section level2">
<h2>2013</h2>
<ul>
<li>Woo, S, Zhang, X, Sauteraud, R, Robert, F. and Gottardo, R. PING: An R/Bioconductor package for nucleosome positioning using next-generation sequencing data (2013). Bioinformatics, 2013;29(16):2049-50.</li>
<li>Finak, G., McDavid A., De Rosa, S., Chattopadhyay, P., Dominguez, M., Roederer, M. and Gottardo, R. Mixture Models for Single-Cell Assays with Application to Vaccine Studies (2013). Biostatistics, 2014;85(3):277-86.</li>
<li>Greg C. Imholte, Renan Sauteraud, Bette Korber, Robert T. Bailer, Ellen T. Turk, Georgia Tomaras, Shauna Shen, John Mascola, Richard Koup, David Montefiori, and Raphael Gottardo. A computational framework for the analysis of peptide microarray antibody binding data with application to HIV vaccine profiling (2013). Journal of Immunological Methods, 2013;395(1-2):1-13.</li>
<li>Dominguez,MH, Chattopadhyay, PK, Ma, S, Lamoreaux, L, McDavid, A, Finak, G, Gottardo, R, Koup, RA, Roederer, M. Highly Multiplexed Quantitation of Gene Expression on Single Cells. Journal of Immunological Methods, 391(1-2): 133–145</li>
<li>McDavid, A, Finak, G, Chattopadyay, PK, Dominguez, M, Lamoreaux, L, Ma, SS, Roederer, M, and Gottardo, R. Data exploration, quality control and testing in single-cell qPCR-based gene expression experiments. Bioinformatics, 29(4): 461-467</li>
<li>Aghaeepour, N, Finak, G, The FlowCAP Consortium, The DREAM Consortium, Hoos, H, Mosmann, TR, Brinkman, R, Gottardo, R, and Scheuermann, RH. Critical assessment of automated flow cytometry data analysis techniques. Nature Methods, 10: 228–238</li>
<li>Casazza, Joseph P; Bowman, Kathryn; Adzaku, Selorm; Smith, Emily C; Enama, Mary E; Bailer, Robert T; Price, David A; Gostick, Emma; Gordon, Ingelise; Ambrozak, David R; Nason, Martha; Roederer, Mario; Andrews, Charla A; Maldarelli, Frank M; Wiegand, Ann; Kearney, Mary F; Persaud, Deborah; Ziemniak, Carrie; Gottardo, Raphael; Ledgerwood, Julie E; Graham, Barney S; Koup, Richard A. Therapeutic Vaccination Expands and Improves the Function of the HIV-specific Memory T Cell Repertoire. Journal of Infectious Diseases 2013;207(12):1829-40</li>
<li>Susan Zolla-Pazner, Timothy Cardozo, Nicos Karasavvas, Raphael Gottardo, Constance Williams, Daryl E Morris, Georgia Tomaras, Mangala Rao, Erik Billings, Phillip Berman, Xiaoying Shen, Charla Andrews, Robert J O’Connell, Viseth Ngauy, Sorachai Nitayaphan, Mark de Souza, Bette Korber, Richard Koup, Robert T Bailer, John R Mascola, Abraham Pinter, David Montefiori, Barton F Haynes, Merlin L Robb, Supachai Rerks-Ngarm, Nelson L Michael, Peter B Gilbert, Jerome H Kim. Analysis of V2 Antibody Responses Induced in Vaccinees in the ALVAC/AIDSVAX HIV-1 Vaccine Efficacy Trial/ PLoS One 8(1):e53629</li>
</ul>
</div>
<div id="section-3" class="section level2">
<h2>2012</h2>
<ul>
<li>Huang, Y. and Gottardo, R. (2012). Comparability and reproducibility of biomedical data. Briefings in Bioinformatics. G. Finak, W. Jiang, J. Pardo, A. Asare, R. Gottardo (2012) QUAliFiER: An automated pipeline for quality assessment of gated flow cytometry data BMC Bioinformatics DOI: 10.1186/1471-2105-13-252</li>
<li>Nicos Karasavvas, Erik Billings, Mangala Rao, Constance Williams, Susan Zolla-Pazner, Robert Bailer, Richard Koup, Sirinan Madnote, Duangnapa Arworn, Xiaoying Shen, Georgia Tomaras, Jeffrey Currier, Mike Jiang, Craig Magaret, Charla Andrews, Raphael Gottardo, Peter Gilbert, Timothy Cardozo, Supachai Rerks-Ngarm, Sorachai Nitayaphan, Punnee Pitisuttithum, Jaranit Kaewkungwal, Robert Paris, Kelli Greene, Hongmei Gao, Sanjay Gurunathan, Jim Tartaglia, Faruk Sinangil, Bette Korber, David Montefiori, John Mascola, Merlin Robb, Barton Haynes, Viseth Ngauy, Nelson Michael, Jerome Kim, and Mark de Souza (2012). The Thai phase III HIV-1 vaccine trial (RV144) regimen induces antibodies that target the crown of the V2 loop of gp120. AIDS Research and Human Retroviruses, 2012;28(11):1444-57.</li>
<li>Arafat Tayeb, Marie-Pier Scott-Boyer, Aurelie Labbe, Christian Deschepper, Raphael Gottardo (2012). An integrated Bayesian Model for Multivariate eQTL Mapping. Statistical Applications in Genetics and Molecular Biology, 11:1544-6115.</li>
<li>X. Zhang, G. Robertson, S Woo, BG Hoffman, R Gottardo (2012) Probabilistic inference for nucleosome positioning with MNase-based or sonicated short-read data. PLoS ONE. 2012;7(2):e32095.</li>
<li>K. Lo and R. Gottardo (2012). Flexible Mixture Modeling via the Multivariate t Distribution with the Box-Cox Transformation: An Alternative to the Skew t Distribution. Statistics and Computing, 22(1): 33-52.</li>
<li>A. Bashashati, N.A. Johnson, A. Hadj Khodabakhshi , M. Whiteside, H. Zare, D.W. Scott, K. Lo, R. Gottardo, F. Brinkman, J. Connors, G. Slack, R.D. Gascoyne, A. Weng, R.R. Brinkman (2012) B-cells with high side scatter parameter by flow cytometry correlate with inferior survival in diffuse large B cell lymphoma. American Journal of Clinical Pathology: 137 (5): 805-814.</li>
</ul>
</div>
<div id="section-4" class="section level2">
<h2>2011</h2>
<ul>
<li>F. Caron, A. Doucet, and R. Gottardo (2011). On-line changepoint detection and parameter estimation with application to genomic data. Statistics and Computing 22:579–595.</li>
<li>X. Zhang, G. Robertson, M. Krzywinski, K. Ning, A. Droit, S. Jones, and R. Gottardo (2011). PICS: Probabilistic Inference for ChIP-seq. Biometrics: 67 (1): 151-163.</li>
<li>Mercier E, Droit A, Li L, Robertson G, Zhang X, and Gottardo R (2011) An Integrated Pipeline for the Genome-Wide Analysis of Transcription Factor Binding Sites from ChIP-Seq. PLoS ONE 6(2): e16432. <a href="doi:10.1371/journal.pone.0016432" class="uri">doi:10.1371/journal.pone.0016432</a></li>
</ul>
</div>
<div id="section-5" class="section level2">
<h2>2010</h2>
<ul>
<li>Finak, G. and Manuel-Perez, J. and R. Gottardo (2010). Automated transformations for flow cytometry. BMC Bioinformatics, 11:546.</li>
<li>K. Lo and R. Gottardo (2009). Flexible Mixture Modeling via the Multivariate t Distribution with the Box-Cox Transformation: An Alternative to the Skew t Distribution, Statistics and Computing, DOI: 10.1007/s11222-010-9204-1.</li>
<li>J.P. Baudry, A.E. Raftery, G. Celeux, K. Lo and R. Gottardo (2010). Combining Mixture Components for Clustering. Journal of Graphical and Computational Statistics, 19: 332:353.</li>
<li>B. Luke, A. Doucet and R. Gottardo (2010). An Efficient Computational Approach for Prior Sensitivity Analysis and Cross-Validation. Canadian Journal of Statistics, 38: 47 - 64.</li>
<li>A. Droit, C. Cheung, and R. Gottardo (2010). rMAT – an R/Bioconductor package for analyzing ChIP-chip experiments, Bioinformatics, 26:678-679.</li>
</ul>
</div>
<div id="section-6" class="section level2">
<h2>2009</h2>
<ul>
<li>Gottardo, R., Brinkman, R.R., Luta, G. and Wand, M.P. (2009). Recent bioinformatics advances in the analysis of high throughput flow cytometry data. Editorial article. Advances In Bioinformatics, <a href="doi:10.1155/2009/461763" class="uri">doi:10.1155/2009/461763</a></li>
<li>G. Finak, A. Bashashati, R. Brinkman, R. Gottardo (2009). Merging Mixture Components for Cell Population Identification in Flow Cytometry. Advances in Bioinformatics, <a href="doi:10.1155/2009/247646" class="uri">doi:10.1155/2009/247646</a></li>
<li>K. Lo, F. Hahne, R. Brinkman and R. Gottardo (2009). flowClust: a Bioconductor package for automated gating of ﬂow cytometry data. BMC Bioinformatics, 10: R145.</li>
<li>R. Gottardo and A. E. Raftery (2009) Bayesian robust variable and transformation selection: A unified approach, Canadian Journal of Statistics, 37: 1-20.</li>
<li>R. Gottardo (2009). Statistical methods for ChIP experiments on microarrays. Humana Press, vol. 567. This chapter will be part of a Volume in the Methods in Molecular Biology series on “Chromatin Immuno-precipitation Assays” edited by Phillipe Collas and published by Humana Press.</li>
<li>R. Gottardo (2009). Bayesian Robust Inference for Differential Gene Expression (In Press). This chapter will be part of a book on “Bayesian Modeling in Bioinformatics” edited by Dipak Dey, Samiran Ghosh and Banni Mallick, and published by Taylor and Francis.</li>
<li>A. Bashashati, K. Lo, R. Gottardo, R. Gascoyne, A. Weng, R. Brinkman (2009). A Pipeline for Automated Analysis of Flow Cytometry Data: Preliminary Results on Lymphoma Sub-Type Diagnosis, IEEE Engineering in Medicine and Biology Society (EMBC’09).</li>
<li>X. Chen and R. Gottardo. An MM-Based Optimization Algorithm for Sparse Linear Modeling on Microarray Data Analysis, 3rd International Conference on Bioinformatics and Biomedical Engineering. ICBBE 2009.</li>
</ul>
</div>
<div id="section-7" class="section level2">
<h2>2008</h2>
<ul>
<li>V. T. Chu, R. Gottardo, A. E. Raftery, R. E. Bumgarner and K. Y. Yeung (2008). MeV+R: Using MEV as a GUI for Bioconductor application. Genome Biology 9:R118.</li>
<li>R. Gottardo, A. E. Raftery (2008). Markov chain Monte Carlo with mixtures of singular distributions. Journal of Computational and Graphical Statistics 17:917-943.</li>
<li>K. Lo, R. Brinkman and R. Gottardo (2008). Automated gating of ﬂow cytometry data via robust model based clustering. Cytometry A, 73:321-332.</li>
<li>R. Gottardo, Li, W., Liu, S. and Johnson, E. (2008). A flexible and powerful Bayesian hierarchical model for ChIP-chip experiments. Biometrics, 64:468-78.</li>
</ul>
</div>
<div id="section-8" class="section level2">
<h2>2007</h2>
<ul>
<li>K. Lo and R. Gottardo (2007). Flexible empirical Bayes models for differential gene expression. Bioinformatics 23: 328.</li>
</ul>
</div>
<div id="section-9" class="section level2">
<h2>2006</h2>
<ul>
<li>W. E. Johnson, Li, W., Meyer, C. A., Gottardo, R., Carroll, J. S., Brown, M., and Liu, X. S. (2006). Model-based analysis of tiling-arrays for ChIP-chip. PNAS 103:12457-12462.</li>
<li>R. Gottardo, A. E. Raftery, K. Y. Yeung and R. Bumgarner (2006). Quality Control and Robust Estimation for cDNA Microarrays with Replicates. Journal of the American Statistical Association, 101:30-40.</li>
<li>R. Gottardo, J. Besag, M. Stephens and A. Murua (2006). Probabilistic segmentation and intensity estimation for microarray images. Biostatistics, 7:85-99.</li>
<li>R. Gottardo, A. E. Raftery, K. Y. Yeung and R. Bumgarner (2006). Bayesian robust inference for differential gene expression in cDNA microarrays with multiple samples. Biometrics, 62:10-18.</li>
</ul>
</div>
<div id="section-10" class="section level2">
<h2>2004</h2>
<ul>
<li>J. F. Challacombe, A. Rechtsteiner, R. Gottardo, L. M. Rocha, E. P. Browne, T. Shenk, M. R. Altherr, and T. S. Brettin (2004). Evaluation of the host transcriptional response to human cytomegalovirus infection. Physiol. Genomics, 18:51-62.</li>
</ul>
</div>
<div id="section-11" class="section level2">
<h2>2003</h2>
<ul>
<li>R. Gottardo, J. A. Pannucci, C. R. Kuske and T. Brettin (2003). Statistical analysis of microarray data: A Bayesian approach. Biostatistics, 4:597-620.</li>
<li>G. Xie, C. A. Bonner, T. Brettin, R. Gottardo, N. O. Keyhani and R. A. Jensen (2003). Lateral gene transfer and ancient paralogy of operons containing redundant copies of tryptophan-pathway genes in Xylella species and in heterocystous cyanobacteria. Genome Biology, 4:R14.</li>
</ul>
</div>

<hr>
<p>Copyright &copy; 2017 <a href="http://www.rglab.org/">rglab</a>. All right reserved.</p>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
