<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>RGLAB::RESEARCH</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>
<link href="site_libs/font-awesome-4.5.0/css/font-awesome.min.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 66px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 71px;
  margin-top: -71px;
}

.section h2 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h3 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h4 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h5 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h6 {
  padding-top: 71px;
  margin-top: -71px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">RGLAB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-pencil"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="software.html">
    <span class="fa fa-laptop"></span>
     
    Software
  </a>
</li>
<li>
  <a href="members.html">
    <span class="fa fa-user"></span>
     
    Members
  </a>
</li>
<li>
  <a href="contact.html">
    <span class="fa fa-envelope"></span>
     
    Contact
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://twitter.com/raphg">
    <span class="fa fa-twitter"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/RGLab">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">RGLAB::RESEARCH</h1>

</div>


<blockquote>
<h2 id="what-do-we-do">What do we do?</h2>
<p>Our main research interest is the development of cutting edge statistical methods and software tools for the analysis of high throughput biological data with an emphasis on immunology and vaccine research. Please use the tab navigation below to learn more about specific active research projects in the lab.</p>
</blockquote>
<div id="section" class="section level2 tabset">
<h2></h2>
<div id="cytometry" class="section level3">
<h3>Cytometry</h3>
<p>We do research on new statistical methods for analyzing flow cytometry data, including methods for determining vaccine positivity, automated and semi-automated gating and data analysis pipelines, as well as quality assurance, quality control, and visualization.</p>
<div id="flowviz-visualization-with-conditioning-on-study-variables" class="section level4">
<h4>FlowViz Visualization With Conditioning on Study Variables</h4>
<p><img src="site_libs/img/flowViz.png" style="width: 500px;"/></p>
<p>We have extended <a href="https://github.com/RGLab/flowViz">flowViz</a> to allow for visualization of large data sets of manually gated flow data with associated study metadata using the standard R formula interface. Data exploration and quality control are greatly simplified with the new API.</p>
</div>
<div id="comparison-of-mimosa-against-other-methods-for-vaccine-positivity-calls" class="section level4">
<h4>Comparison of MIMOSA against other methods for vaccine positivity calls</h4>
<p><img src="site_libs/img/MIMOSA_ROC.png" style="width: 500px;"/></p>
<p>We are developing statistical methods for downstream analysis of flow cytometry assay data. MIMOSA is a new method for identifying vaccine responders from ICS (intracellular cytokine staining) data. The figure shows a comparison of the sensitivity and specificity of MIMOSA compared to other methods for determining vaccine response.</p>
</div>
</div>
<div id="peptide-microarrays" class="section level3">
<h3>Peptide microarrays</h3>
<p>Our group researches advanced statistical modeling techniques to improve the power to detect antibody epitopes using peptide microarray experiments. We aim to develop flexible, robust methodology that shares relevant information between subjects while accommodating aspects of experiments that are unique to immunological applications. We are also developing a unified software environment to streamline data analysis from preprocessing up through detection and visualization. Our methodology is currently being applied to many vaccine trials including human and non-human primate studies.</p>
<p>The two figures below show an application of our analysis and visualization pipeline applied to the RV144 pilot microarray data published in <a href="http://online.liebertpub.com/doi/full/10.1089/aid.2012.0103">AIDS Research and Human Retroviruses</a>.</p>
<div id="rv144-v2-loop-aggregate-response" class="section level4">
<h4>RV144 V2 loop aggregate response</h4>
<p><img src="site_libs/img/V2-RV144.png" style="width: 500px;"/></p>
<p>We can observe a strong V2 response in the vaccine group and no response in the placebo group.</p>
</div>
<div id="rv144-v2-loop-subtype-specific-responses" class="section level4">
<h4>RV144 V2 loop subtype specific responses</h4>
<p><img src="site_libs/img/V2-RV144-subtype.png" style="width: 500px;"/></p>
<p>The strongest response is observed in CRF01 whereas B is the smallest.</p>
</div>
</div>
<div id="single-cell-expression" class="section level3">
<h3>Single-cell expression</h3>
<p>Although cells sorted using well-established surface markers may appear homogeneous, mRNA expression of other genes within these cells can be heterogeneous and could further characterize cell poly-functionality. New technologies based on microfluidic arrays combined with multiplexed polymerase chain reactions (PCR) can now be used to perform thousands of PCRs in a single device, enabling simultaneous, high-throughput gene expression measurements at the single-cell level across hundreds of cells and genes. While classic gene expression microarrays sum the expression from many individual cells, the intrinsic stochastic nature of biochemical processes results in relatively large cell-to-cell gene expression variability.This heterogeneity may carry important information, thus single cell expression data should not be analyzed in the same fashion as cell-population level data. We are currently developing statistical methods and tools for the analysis of single cell gene expression data (e.g. <a href="http://arxiv.org/abs/1208.5809">MIMOSA</a>, <a href="http://arxiv.org/abs/1210.1226">SingleCellAssay</a>).</p>
<div id="single-cell-gene-expression-and-positivity" class="section level4">
<h4>Single-Cell Gene Expression and Positivity</h4>
<p><img src="site_libs/img/Fluidigm.png" style="width: 500px;"/></p>
<p>Subject-specific differences as well as stimulation-specific common patterns in single-cell gene expression are observed between differentially stimulated, tetramer-sorted cell subpopulations. The data was analyzed using our new MIMOSA (Mixture Models for Single-Cell Assays) tool.</p>
</div>
<div id="concordance-of-single-cell-and-hundred-cell-gene-expression-experiments" class="section level4">
<h4>Concordance of Single-Cell and Hundred-Cell Gene Expression Experiments</h4>
<p><img src="site_libs/img/concordcomp.png" style="width: 500px;"/></p>
<p>The concordance between single-cell gene expression experiments and the average per-cell expression from 100-cell gene expression experiments is shown for three different data sets (columns) when unexpressed genes are treated as missing observations (second row) or as exact zeros (top row). Filtering of outlier values is applied (last row) improving concordance between the experiments. Treating unexpressed genes as zeros rather than missing observations improves the concordance, indicating that these genes are truly not expressed rather than undetectable through this technology.</p>
</div>
</div>
<div id="immunologyvaccinology" class="section level3">
<h3>Immunology/Vaccinology</h3>
<p>We collaborate with bench scientists and clinicians throughout multiple networks including the <a href="http://www.hvtn.org/">HIV Vaccine Trials Network</a>, the <a href="http://www.mtnstopshiv.org/">Microbicide Trials Network</a>, the <a href="http://www.immunetolerance.org/">Immune Tolerance Network</a> and the <a href="http://www.immuneprofiling.org/">Human Immunology Project Consortium</a>. In particular we develop and implement tools and methods for data processing and analysis of immunological assays used for measuring vaccine immunogenicity (e.g., ICS, multiplex bead arrays), increasingly involving higher dimensional assays (e.g., mass cytometry for assessing high-dimensional functions of single cells), and to provide statistical validation and standardization of assays. Using these tools, our aim is to use system approaches to predict how the immune system will respond to a particular infection of vaccination.</p>
<p>Here are some of the applications we are currently working on:</p>
<div id="epitope-mapping" class="section level4">
<h4>Epitope mapping</h4>
<p>In collaboration with the <a href="http://www.hvtn.org/">HVTN</a> we are testing new methods for positivity calls (<a href="http://arxiv.org/abs/1208.5809">MIMOSA</a>) on other assays, including epitope mapping and EliSpot.</p>
</div>
<div id="immune-correlates" class="section level4">
<h4>Immune correlates</h4>
<p>We are currently working on identifying immune response biomarkers that correlate with vaccine efficacy (e.g. protection/infection) for the RV144 and RTS’S trials.</p>
</div>
<div id="poly-functionality" class="section level4">
<h4>Poly-functionality</h4>
<p>In collaboration with researchers from the <a href="http://www.hvtn.org/">HVTN</a>, we are working on utilizing high dimensional technologies that measure multiple functions (e.g. genes, cytokines) from individual cells to characterize cell poly-functionality in response to vaccination. The working hypothesis is that cells producing multiple functions or certain combinations of functions are more beneficial to a clinical outcome such as control or prevention of infection.</p>
</div>
</div>
<div id="epigenomics" class="section level3">
<h3>Epigenomics</h3>
<p>For the past decade, microarrays have been the platform of choice for various high-throughput biological studies. In fact, our group has significantly contributed to the development of statistical methods for microarray data. Recent improvements in the efficiency, quality, and cost of genome-wide sequencing are prompting biologists to rapidly abandon microarrays in favor of so-called next-generation sequencers. Next-generation sequencing technologies, capable of producing tens of millions of sequence reads during each instrument run, have already been applied for studying genome-wide transcription levels (RNA-Seq), transcription factor binding sites and nucleosome positioning (ChIP-Seq), and DNA methylation status. While sequencing-based studies have been promoted as overcoming longstanding limitations of microarray-based studies, these new biotechnologies raise similar as well as novel statistical and computational challenges. These sequencing technologies generate massive data sets (much larger than with microarrays), and there is a pressing need for statistical and computational tools to make sense of these data. To this end, our group has developed several methods for the analysis of ChIP-Seq and MNase-Seq data. The Figures below illustrate our analysis pipeline using <a href="http://www.bioconductor.org/packages/devel/bioc/html/PICS.html">PICS</a>, <a href="http://www.bioconductor.org/packages/devel/bioc/html/rGADEM.html">rGADEM</a>, <a href="http://www.bioconductor.org/packages/devel/bioc/html/MotIV.html">MotIV</a> and <a href="http://www.bioconductor.org/packages/devel/bioc/html/PING.html">PING</a>.</p>
<div id="pics-increases-spatial-resolution-of-binding-event-detection" class="section level4">
<h4>PICS increases spatial resolution of binding event detection</h4>
<p><img src="site_libs/img/RG_SW_bwfigure2.png" style="width: 500px;"/></p>
<p>Transcription factor binding sites identified by PICS, iSeq, BayesPeak and MACS. PICS identifies two binding events in this region while other three methods identify these two events as one broad enriched region.</p>
</div>
<div id="ping-identifies-weakly-positioned-nucleosomes." class="section level4">
<h4>PING identifies weakly positioned nucleosomes.</h4>
<p><img src="site_libs/img/RG_SW_bwfigure4.png" style="width: 500px;"/></p>
<p>Expected Nucleosome positioning in Chr1 for 149000-153000 genomic region using PING and TpF. PING can identify nucleosome positioning with weak signal compared to TpF.</p>
</div>
</div>
<div id="statistical-computing" class="section level3">
<h3>Statistical computing</h3>
<p>Data resulting from novel high throughput technologies have lead to novel statistical problems and challenges. In addition, data are being accumulated more quickly than they can actually be analyzed. As a consequence, it is essential that analytical tools and statistical methods evolve in parallel with the assay technologies and datasets. Our group has developed many statistical methods for the analysis of high dimensional high throughput datasets including mixture models, sequential Monte Carlo, Markov chain Monte Carlo, and change point models (See examples below). We also implement state-of-the-art software for the analysis of large datasets using parallel programming (OpenMP, GPU) and efficient data formats (NetCDF, HDF5).</p>
<div id="change-point-modeling-of-dna-tiling-arrays" class="section level4">
<h4>Change-point modeling of DNA tiling arrays</h4>
<p><img src="site_libs/img/tiling-array-SMC.png" style="width: 500px;"/></p>
<p>Segmentation results for part of chromosome 1 for the yeast genome expression data using our SMC algorithm and least-squares change point model estimation. For the top graph, the MAP estimate is displayed with transcript segments (green background), non-transcript segments (white background) and black segments for the segment intensity levels. For both top and bottom graphs, segments boundaries are represented with green vertical lines. Transcript annotations are shown below with red rectangles representing coding sequences and black segments representing TF binding sites. More precise estimation of known transcript boundaries can be obtained with <a href="http://www.springerlink.com/content/3880054343602gw8/">our approach</a>.</p>
</div>
<div id="lasso-type-regularization-path-plots-smc-vs.mcmc" class="section level4">
<h4>Lasso-type regularization path plots SMC vs. MCMC</h4>
<p><img src="site_libs/img/SMC-MCMC.png" style="width: 500px;"/></p>
<p>Plots using MCMC and SMC for fixed computational time of 5 min. The MCMC plot is a lot more noisy than the SMC version (<a href="http://www.springerlink.com/content/3880054343602gw8/">our approach</a>).</p>
</div>
</div>
</div>

<hr>
<p>Copyright &copy; 2017 <a href="http://www.rglab.org/">rglab</a>. All right reserved.</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
